tests/extract-css.js

```diff
@@ -245,17 +241,50 @@ const tests = [
          <p>The distance is measured between the initial position and the intersection of the ray with the box. If the initial position is not within the box, the distance is 0.</p>
        </dd></dl>
       </dd></dl>`,
-    propertyName: "valuespaces",
-    css: {
-      "<size>": {
-        "prose": "Decides the path length used when offset-distance is expressed as a percentage, using the distance to the containing box. For <size> values other than sides, the path length is independent of <angle>. It is defined as: <size> = [ closest-side | closest-corner | farthest-side | farthest-corner | sides ]"
```

Comment:
Yes in the sense that changing the `<p>` into a `<pre class="prod">` ([line 225](https://github.com/w3c/reffy/pull/1117/files#diff-4bf71a2be3248cbd91bdaf47cec5144b9a708bcc259c350244eda272fd8886e6R225)) means the extraction code now has a proper `value` property to set. When there is a `value` property, the code does not (and did not, no change here) attempt to extract prose.

Prose extraction is a good topic on its own. As currently done, using raw text, I think it is only useful for debugging (e.g. understanding where a definition comes from).  For rendering, tools would probably prefer HTML and not raw text. It could be interesting to rather return HTML content for the prose, and to include the prose more systematically. Regardless of the format, it is hard to guarantee that the prose is meaningful though.